src/views/summary/CustomOfflineChart.js

```diff
@@ -75,23 +103,101 @@ function CustomOfflineChart({ chartType, barTitle, height, datasetName, dataObje
         function createBarChart() {
             const data = [];
 
+            // See dataObject type
             const categories = Object.keys(dataObject).map((key) => {
                 data.push(dataObject[key]);
                 return key;
             });
+
             setChartOptions({
                 credits: {
                     enabled: false
                 },
+                colors: [theme.palette.primary.dark],
                 series: [{ data, colorByPoint: true, showInLegend: false }],
-                xAxis: { categories }
+                xAxis: { categories },
+                tooltip: {
+                    useHTML: true,
+                    formatter: function () {
+                        var dataSum = 0,
+                            pcnt;
+
+                        this.series.points.forEach(function (point) {
+                            dataSum += point.y;
+                        });
+
+                        pcnt = (this.y / dataSum) * 100;
+                        return '<b>' + this.point.category + '</b>' + '<br> - ' + this.y + ' <br> - ' + Highcharts.numberFormat(pcnt) + '%';
+                    }
+                }
             });
         }
 
-        if (chartType === 'pie') {
+        /*
+         * Create Stacked Bar chart from props
+         */
+
+        function createStackedBarChart() {
+            const data = new Map();
+            const categories = [];
+            let i = 0;
+
+            Object.keys(dataObject).map((key) => {
+                categories.push(key);
+                Object.keys(dataObject[key]).map((cohort) => {
+                    if (data.has(cohort)) {
+                        data.get(cohort).splice(i, 1, dataObject[key][cohort]);
+                    } else {
+                        data.set(cohort, new Array(Object.keys(dataObject).length).fill(0));
+                        data.get(cohort).splice(i, 1, dataObject[key][cohort]);
+                    }
+                });
+                i++;
+            });
```

Comment:
It's more idiomatic to use forEach() here -- `.map()` is for when you care about the return value. It also lets us use the built-in iterator, like so:
```suggestion

            Object.keys(dataObject).forEach((key, i) => {
                categories.push(key);
                Object.keys(dataObject[key]).forEach((cohort) => {
                    if (!data.has(cohort)) {
                        data.set(cohort, new Array(Object.keys(dataObject).length).fill(0));
                    }
                    data.get(cohort).splice(i, 1, dataObject[key][cohort]);
                });
            });
```
I haven't yet tested the above, but something along these lines might be easier to read.